tests/TypesService.test.ts

```diff
@@ -43,7 +43,10 @@ describe('relative import remapping', () => {
           ),
           ts.factory.createStringLiteral('./foo'),
         )
-        const transformed = Types.rewriteRelativeImportSpecifier(rewrite)(testNode)
+        const transformed = Types.rewriteRelativeImportSpecifier(
+          rewrite,
+          () => false,
+        )(testNode)
```

Comment:
It would be good to add a test case for index exports

Reply:
I agree, but that requires more extensive setup than I know how to do with `ts.factory`, since its indexness isn't a property on the parsed node itself. It only exists in the context of a full project.

It'd probably be good to have E2E tests, though. I can try to set some up

Reply:
That'd be rad, but no worries if not